### ISSUE TITLE
FindWidget always has aria-hidden="false" even when hidden

### ISSUE THREAD

**Issue by jayspadie** [Microsoft/monaco-editor#450]
monaco-editor version: 0.8.3
Browser: Any
OS: Windows 10

During accessibility testing, we found that screen readers (including NVDA on Chrome) will read the Find Widget buttons even when the control is not visible.  This appears to be a bug in [FindWidget.ts](https://github.com/Microsoft/vscode/blob/a3dba6ed08e32f175ca0e55fc667bd8c7a861e00/src/vs/editor/contrib/find/browser/findWidget.ts#L339) (“editor-widget find-widget”) where aria-hidden always equals "false” even when the Find dialog is closed and not visible.  I believe the fix would be to also add dom.setAttribute(“aria-hidden”, “true”) when removing the visible class here on [FindWidget.ts](https://github.com/Microsoft/vscode/blob/a3dba6ed08e32f175ca0e55fc667bd8c7a861e00/src/vs/editor/contrib/find/browser/findWidget.ts#L339).  The issue is demonstrated below with NVDA reading the Find buttons that aren't actually visible.

![image](https://cloud.githubusercontent.com/assets/5246869/26651155/ca9e18a4-4619-11e7-9381-85d872565cd8.png)

**Comment by alexandrudima** [comment]
Fixed with the fix for Microsoft/vscode#27831
